wavelength-plugins/splatoon.js

```diff
@@ -97,17 +97,16 @@ exports.commands = {
 			on: function (target, room, user) {
 				if (!this.can("ban", null, room)) return this.errorReply(`You must be a Room Moderator or higher to use this command.`);
 				if (room.id !== "splatoon") return this.errorReply(`This command only works in the Splatoon room.`);
-				let targets = target.split(",");
-				for (let u in targets) targets[u] = targets[u].trim();
-				if (!targets[1]) return this.parse("/splatoonhelp");
+				let [team1, team2] = target.split(",").map(p => p.trim());
+				if (!team2) return this.parse("/splatoonhelp");
```

Comment:
`if (!(team1 && team2))` to ensure that both `team1` and `team2` are present and are truthy (not empty strings).

Reply:
oh